src/extension.ts

```diff
@@ -36,9 +36,11 @@ function handleDiagnosticsForActiveEditor(document: vscode.TextDocument, previou
             lineText = lineText.split("//")[0];
             // trim the line text
             lineText = lineText.trim();
-            if (!lineText.endsWith(";")) {
-                return;
-            }
+
+            // is it possible to remove this condition to support TSX?
```

Comment:
Just a suggestion:
This condition prevents the sound from being played when there is error in TSX (usually there is no `;` in TSX). So is it possible to remove it?

Reply:
Thank you for your suggestion. The condition you mentioned prevents premature sound alerts for type errors. Without it, developers might get false positives, disrupting their workflow.

I understand this condition might sometimes prevent necessary alerts. I'm open to ideas. What are your thoughts?

Reply:
No thoughts for now. Need to deep dive into the code and find a workaround. Maybe it's better to ignore it in this PR and do it as a feature. I'll uncomment the code.